nucypher/characters/base.py

```diff
@@ -123,21 +133,18 @@ def __init__(self,
                 raise ValueError(f"Cannot init federated-only character with {registry or provider_uri}.")
         self.federated_only: bool = federated_only
 
+        ##########################################
+
         #
-        # Powers
+        # Keys & Powers
         #
 
-        # Derive powers from keyring
-        if keyring_root and keyring:
-            if keyring_root != keyring.keyring_root:
-                raise ValueError("Inconsistent keyring root directory path")
         if keyring:
             keyring_root, checksum_address = keyring.keyring_root, keyring.checksum_address
```

Comment:
If a keyring is passed, it has an associated checksum address and related keys.  If a different address is passed that does not match the keyring, then we can raise an exception I think since we cannot know what address the caller was trying to use.

Reply:
✔️